etna/articles/models.py

```diff
@@ -144,6 +144,8 @@ class ArticlePage(
     # DataLayerMixin overrides
     gtm_content_group = "stories"
 
+    title_label = "THE STORY OF"
```

Comment:
Would just attribute name `label` be sufficient - since its representing a name for that specific page?

Reply:
I didn't want this to clash with anything internal for Wagtail. I think `title_label` is best, as it implies it is a label for the title, but happy to change if you think :)

Reply:
>  didn't want this to clash with anything internal for Wagtail. 

It does not clash with anything internal for Wagtail.

> I think title_label is best, as it implies it is a label for the title, but happy to change if you think :)

I'm not going to step on this :)